README.md

```diff
@@ -351,9 +354,43 @@ Predictions made on an ensemble of models will return the average of the individ
 
 If installed from source, `chemprop_predict` can be replaced with `python predict.py`.
 
-### Epistemic Uncertainty
+### Uncertainty Estimation
 
-One method of obtaining the epistemic uncertainty of a prediction is to calculate the variance of an ensemble of models. To calculate these variances and write them as an additional column in the `--preds_path` file, use `--ensemble_variance`. If this flag is used with a spectra prediction, it will instead return the average pairwise SID comparison of the different ensemble predictions.
+The uncertainty of predictions made in Chemprop can be estimated by several different methods. Uncertainty estimation is carried out alongside model value prediction and reported in the predictions csv file when the argument `--uncertainty_method <method>` is provided. If no uncertainty method is provided, then only the model value predictions will be carried out. The available methods are:
+
+* `ensemble` For a prediction using an ensemble of models. Returns the variance of predictions made by each of the ensemble submodels. Ensemble variance can be used with any dataset type, but the results are only usable for calibration or evaluation with regression datasets.
+* `dropout` Intended for use with a single model and not an ensemble. This method uses dropout to generate a virtual ensemble of models and reports the ensemble variance of the predictions. The number of models generated and the probability of dropout can be changed using `--uncertainty_dropout_p <float>` and `--dropout_sampling_size <int>`, respectively.
```

Comment:
Maybe worth adding a note here that this can be used regardless of how the model was trained, and that --uncertainty_dropout_p is different from and overrides the --dropout parameter specified during training